### PR TITLE
Update the checkmark icon in org selector

### LINKED ISSUE #16434: Improve the checkmark icon for the selected organization in the org selector
### Problem to solve

Following up from https://github.com/gitpod-io/gitpod/pull/16422, this is a placeholder issue to improve the **checkmark icon** for the selected organization in the org selector. Ignoring the small design details here's the checkmark for light and dark themes we need to replace. Cc @selfcontained 

```svg
// LIGHT THEME
<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" fill="none">
  <path fill="#78716C" fill-rule="evenodd" d="M18.2348 5.8867 7.88699 16.2345l-2.12132-2.1213L16.1135 3.76538l2.1213 2.12132Z" clip-rule="evenodd"/>
  <path fill="#78716C" fill-rule="evenodd" d="m3.88695 8.06069 5.00004 5.00001-2.12132 2.1214-5.00005-5.0001 2.12133-2.12131Z" clip-rule="evenodd"/>
</svg>

// DARK THEME
<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" fill="none">
  <path fill="#E7E5E4" fill-rule="evenodd" d="M18.2348 5.8867 7.88699 16.2345l-2.12132-2.1213L16.1135 3.76538l2.1213 2.12132Z" clip-rule="evenodd"/>
  <path fill="#E7E5E4" fill-rule="evenodd" d="m3.88695 8.06069 5.00004 5.00001-2.12132 2.1214-5.00005-5.0001 2.12133-2.12131Z" clip-rule="evenodd"/>
</svg>
```

| BEFORE | AFTER |
|-|-|
| org-a1 | Navigation |
| org-a2 | Navigation-1 |

### PR DESCRIPTION
**Pull request by Devansu-Yadav**
## Description

- Updates the checkmark icon for the selected organization in the org selector.

## Related Issue(s)

Fixes #16434 

## Release Notes

```release-note
Improve the checkmark icon for the selected organization in the org selector
```

## Build Options:

- [ ] /werft with-github-actions
      Experimental feature to run the build with GitHub Actions (and not in Werft).
- [ ] leeway-no-cache
      leeway-target=components:all
- [ ] /werft no-test
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer Options</summary>

- [ ] with-ee-license
- [ ] with-slow-database
- [ ] with-dedicated-emulation
- [ ] with-ws-manager-mk2
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

File: components/dashboard/src/menu/OrganizationSelector.tsx [hunk 1]
```diff
@@ -251,8 +251,36 @@ const CurrentOrgEntry: FunctionComponent<CurrentOrgEntryProps> = ({ title, subti
                 <span className="text-gray-800 dark:text-gray-300 text-base font-semibold">{title}</span>
                 <span>{subtitle}</span>
             </div>
-            {/* TODO: Replace this with an SVG icon */}
-            <div className="pl-1 font-semibold">&#x2713;</div>
+
+            <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" className="dark:hidden" fill="none">
```
Comment (gtsiolis) [inline components/dashboard/src/menu/OrganizationSelector.tsx:255]: **praise**: Nice!

File: components/dashboard/src/menu/OrganizationSelector.tsx [hunk 2]
```diff
@@ -251,8 +251,36 @@ const CurrentOrgEntry: FunctionComponent<CurrentOrgEntryProps> = ({ title, subti
...
+                />
+                <path
+                    fill="#78716C"
+                    fill-rule="evenodd"
+                    d="m3.88695 8.06069 5.00004 5.00001-2.12132 2.1214-5.00005-5.0001 2.12133-2.12131Z"
+                    clip-rule="evenodd"
+                />
+            </svg>
+
+            <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" className="hidden dark:block" fill="none">
```
Comment (gtsiolis) [inline components/dashboard/src/menu/OrganizationSelector.tsx:270]: **praise**: Nice! Cc @selfcontained for visibility.

**Review by gtsiolis — APPROVED** [review]
UX LGTM, @Devansu-Yadav!

Thanks again for contributing! 🍊

**Comment by Devansu-Yadav** [comment]
@gtsiolis I was wondering if I could pick up #3836 next? 😄

**Pull request merged**